packages/langium/src/test/langium-test.ts

```diff
@@ -556,10 +556,11 @@ export type ValidationHelperOptions = ParseHelperOptions & { failOnParsingErrors
 export function validationHelper<T extends AstNode = AstNode>(services: LangiumCoreServices): (input: string, options?: ValidationHelperOptions) => Promise<ValidationResult<T>> {
     const parse = parseHelper<T>(services);
     return async (input, options) => {
-        const document = await parse(input, {
+        const parseOptions: ValidationHelperOptions = {
             ...(options ?? {}),
-            validation: true
-        });
+        };
+        parseOptions.validation = parseOptions.validation ?? true;
```

Comment:
FYI, the nullish coalescing assignment operator (`??=`) is supported since TypeScript 4.0 and Node 15.0

Reply:
Thanks for the information.
I'm not (yet) expert in typescript